include/ensmallen_bits/callbacks/progress_bar.hpp

```diff
@@ -161,8 +155,8 @@ class ProgressBar
       }
     }
 
-    output << "] " << progress << "% - ETA: " << (size_t) stepTimer.toc() *
-        (epochSize - step + 1) % 60 << "s - loss: " <<
+    output << "] " << progress << "% - ETA: " <<  fmod(stepTimer.toc() *
+        (double)(epochSize - step + 1), 60) << "s - loss: " <<
```

Comment:
`fmod()` will return a `double`; we probably want to use the integer-valued `%`.  Maybe, e.g., `size_t(stepTimer.toc() * (epochSize - step + 1)) % 60`?  (You might want to double check if that works right; I didn't test it.)

Reply:
Hi, @rcurtin I made the necessary changes kindly have a look at it now. Thanks.